gui/app/main.cpp

```diff
@@ -16,12 +18,37 @@
 #include <iostream>
 #include <memory>
 
+#include "batteryinfo.h"
+#include "simple_clock.h"
+
+QObject *clockInstance(QQmlEngine *engine, QJSEngine *scriptEngine) {
```

Comment:
Our preferred naming style is either PascalCase or snake_case for functions

Reply:
Yes sorry. Qt & google style are very conflicting. Hard to wrap my head around.